src/cli/config.go

```diff
@@ -0,0 +1,7 @@
+package cli
+
+// Config defines the configuration values needed by the `cli` package.
+type Config interface {
+	GetBranchAncestryRoots() []string
+	GetChildBranches(string) []string
+}
```

Comment:
This interface is pretty specific to input for `PrintableBranchAncestry`. Thoughts on moving it into the printable file and calling it `BranchAncestryConfig`?

Reply:
I'm all for keeping interfaces small and focused. #1546 implements this suggestion.